src/components/Header/Header.scss

```diff
@@ -10,6 +10,11 @@
   height: 90px;
   padding: 0 $spacing--xLarge;
   background-color: $color--primary;
+  @include breakpoint-xl {
+    max-width: $layout-breakpoint-xl;
+    margin-left:auto;
```

Comment:
Nit: You can also do `margin: 0 auto` here, since it doesn't look like the Header has any other margin here before you set it.